src/components/new-suggestion/areas/review.area.tsx

```diff
@@ -1,16 +1,162 @@
+
+  const saveSuggestion = () => {
+    setShowSpinner(true)
+    insertSuggestion(supabase, suggestion, user!.id)
+      .then((response) => {
+        if (response.error) {
+          handleError()
+          return
+        }
+
+        const suggestionId = response.data.at(0)!.id
+        insertSuggestionInstruments(supabase, mapInstruments(suggestionId))
+          .then((response) => {
+            if (response.error) {
+              handleError()
+              return
+            }
+
+            dispatch(updateNewSuggestion(initialState.suggestion))
+            dispatch(setActiveArea(Area.SongInformation))
+            router.push("/suggestions")
+          })
+          .catch(() => handleError())
+      })
+      .catch(() => handleError())
+      .finally(() => {
+        setShowSpinner(false)
+      })
+  }
+
+  const handleError = () => {
+    setInsertError(true)
+    setShowSpinner(false)
+  }
+
+  const mapInstruments = (suggestionId: string): SuggestionInstrumentDatabaseOperation[] => {
+    return suggestion.instruments.map(({ instrument, description }) => {
+      return { suggestion_id: suggestionId, instrument_id: instrument.id, description: description }
+    })
+  }
+
+  const requiredDataIsPresent = () => {
+    return (
+      suggestion.artist.length > 0 &&
+      suggestion.motivation.length > 0 &&
+      suggestion.title.length > 0 &&
+      suggestion.instruments.length > 0
+    )
+  }
+
+  const btnCss = () =>
+    requiredDataIsPresent()
+      ? "bg-moon-500 cursor-pointer"
+      : "bg-moon-100 cursor-not-allowed hover:bg-moon-100"
+
   return (
     <div data-cy="area-review">
       <h2 className={"area-header"}>Review</h2>
-      <p>Title: {newSuggestion.title}</p>
-      <p>Artist(s): {newSuggestion.artist}</p>
-      <p>Motivation: {newSuggestion.motivation}</p>
-      <p>Link: {newSuggestion.link}</p>
+      {showSpinner ? (
+        <div className={"h-[75vh] text-center"} data-cy="suggestions-spinner">
+          <Spinner size={10} />
+        </div>
+      ) : (
+        <>
+          <div className={"m-2 md:ml-auto md:mr-auto md:max-w-sm"}>
+            <div className={"flex"}>
+              <MusicalNoteIcon className={"h-14 w-14 rounded-md bg-neutral-200 p-2 text-black"} />
+              <div className={"ml-3"}>
+                <span data-cy="review-title">
+                  <p className={"line-clamp-1 font-bold"}>{suggestion.title}</p>
+                </span>
+                <span data-cy="review-artists">
+                  <p className={"line-clamp-1 text-left"}>{suggestion.artist.join(", ")}</p>
+                </span>
+              </div>
+            </div>
+            <p
+              className={"mb-3 mt-3 line-clamp-3 h-12 text-sm leading-4 text-gray-400"}
+              data-cy="review-motivation"
+            >
+              {suggestion.motivation}
+            </p>
+          </div>
+
+          <p className={"mb-4 text-center text-lg font-medium text-moon-200"}>Instruments</p>
```

Comment:
```suggestion
          <p className={"mb-4 text-center text-lg text-moon-400"}>Instruments</p>
```

Color change increases readability and removal of font-medium still creates feeling of lower importance than the review header.